vanilla-framework/vanilla-framework
# Escape hash in code snippet svg

## Done

Escape hash in code snippet svg

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/code-snippet/
- Make sure you can see the icons in Firefox

This here https://bugzilla.mozilla.org/show_bug.cgi?id=1016316 gives more of an insight into the issue
